src/app/components/tree-view/index.html

```diff
@@ -86,7 +90,7 @@ <h3>Split view</h3>
       <h3>Expand/collapse nodes</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-            <p>Clicking the chevron button expands or collapses the node.</p>
+            <p>The expand/collapse button expands and collapses the node.</p>
```

Comment:
I changed this back to `expand/collapse button` because that's what we call it in the Anatomy section. If we want to use `chevron button` instead, I think we should adjust all instances of the terminology.

Reply:
leaving it as expand/collapse button seems ok